Patch status: obsolete?

chromium has removed some GLX code which Qt has temporarily(?)
restored in [1]. This however seems to misses some headers and
only works with some configurations and fails with others[2].
So add the headers here as a quick fix for now.

Update: need to be rechecked, believe gl_context.h still "should"
include gl_surface given it uses gl::GLSurface but unclear if this
still results in a failure.

[1] https://github.com/qt/qtwebengine-chromium/commit/f2b407a61bea122d18a012f0049ba193725f0461
[2] https://bugs.gentoo.org/928508
--- a/src/3rdparty/chromium/ui/gl/gl_context.h
+++ b/src/3rdparty/chromium/ui/gl/gl_context.h
@@ -22,4 +22,5 @@
 #include "ui/gl/gl_implementation_wrapper.h"
 #include "ui/gl/gl_share_group.h"
+#include "ui/gl/gl_surface.h"
 #include "ui/gl/gl_state_restorer.h"
 #include "ui/gl/gpu_preference.h"
